### DISCUSSION TITLE
What does noice use for this? (I'm not sure what to call it)

### DISCUSSION THREAD

**Discussion by Radrahil** [folke/noice.nvim#766] [Q&A]
![image](https://github.com/folke/noice.nvim/assets/54015030/ee1fceda-27de-4b10-8ebf-e99c71482c03)
I'm referring to the Metadata fetching rust analyzer part in the bottom right corner. What does noice use for this? It's the only part of it I want.

**Comment by gustavoalves23** [comment]
I believe is what is called the "mini" view in noice, but this doesnt look like the mini.nvim notify plugin. I want it too + the "mini" view notification.

**Reply by folke** [reply]
That's the lsp-progress unctionality using the mini view. The mini view has nothing to do with the mini.notify plugin
